CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 
 Unreleased
 ==========
+
+1.2.0 (2022-10-05)
```

Comment:
If this release only includes bugfixes, I think this should only be a patch release, based on the descriptions on https://semver.org/
```suggestion
1.1.2 (2022-10-05)
```